README.md

```diff
@@ -236,479 +236,900 @@ The base class sets defaults used by other defined types, such as `tomcat::insta
 
 #####`catalina_home`
 
-Specifies the default root directory of the Tomcat installation. Valid options: a string containing an absolute path. Default: '/opt/apache-tomcat'.
+Specifies the default root directory of the Tomcat installation.
+
+Valid options: a string containing an absolute path.
+
+Default value: `/opt/apache-tomcat`.
 
 #####`group`
 
-Specifies a default group to run Tomcat as. Valid options: a string containing a valid group name. Default: 'tomcat'.
+Specifies a default group to run Tomcat as.
+
+Valid options: a string containing a valid group name.
+
+Default value: `tomcat`.
 
 #####`install_from_source`
 
-Specifies whether to default to installing Tomcat from source. Valid options: 'true' and 'false'. Default: 'true'.
+Specifies whether to default to installing Tomcat from source.
+
+Valid options: `true` and `false`.
+
+Default value: `true`.
 
 #####`manage_group`
 
-Determines whether defines should default to creating the specified group, if it doesn't exist. Uses Puppet's native [`group` resource type](https://docs.puppetlabs.com/references/latest/type.html#group) with default parameters. Valid options: 'true' and 'false'. Default: 'true'.
+Determines whether defines should default to creating the specified group, if it doesn't exist. Uses Puppet's native [`group` resource type](https://docs.puppetlabs.com/references/latest/type.html#group) with default parameters.
```

Comment:
I'm still seeing defines (the noun, that is) instead of defined types. I think I failed at Github review before, so I'm not sure whether you saw my earlier comment. (Or if maybe I'm reviewing an old version of this doc now!)